### PR TITLE
Separate pages into different files

### PR DESCRIPTION
**Pull request by BatesJernigan**
make different pages into separate files to be sourced into ui.r, change some names of variables

### CODE REVIEW

**Comment by BatesJernigan** [comment]
@yeabry @ctayl138

**Comment by BatesJernigan** [comment]
@ctayl138 I looked at your revisions to the state average debt. Looking very good! Nice work!

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# History files
```
Comment (BatesJernigan) [inline .gitignore:1]: I pulled this file from [Github's gitignore repo](https://github.com/github/gitignore). It's a great resource for new projects in unfamiliar languages

File: ui.R [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+library(mapproj)
+
+my_autocomplete_list <- c("Web Developer", "Software Developer", "Finance", "Biologist",
+                          "Chemist", "Physist", "Doctor", "Engineer", "Manager")
+
+# Define UI for application that draws a histogram
+shinyUI(
+  navbarPage(
+    "Student Loan Data",
+    source('introduction.r', local=TRUE)$value,
```
Comment (BatesJernigan) [inline ui.R:13]: the `$value` is what removed the TRUE or FALSE value from the bottom of the page for anyone who was interested.

**Comment by ctayl138** [comment]
Ok cool, this looks like a better way of going about this, i'll checkout and work in this model from now on.

**Pull request merged**